libraries/chaindb/CMakeLists.txt

```diff
@@ -0,0 +1,111 @@
+# Defines ChainBase library target.
+cmake_minimum_required( VERSION 3.12 )
+project( ChainBase )
```

Comment:
Don't think we want `project` anymore or `cmake_minimum_required`.

Reply:
Actually quite a bit of this file should probably be deleted.

Reply:
https://github.com/AntelopeIO/spring/pull/1449/commits/f2fba95810892649598c7f731f6b0fc0f5543461